django/contrib/admin/checks.py

```diff
@@ -987,9 +988,14 @@ def _check_actions_uniqueness(self, obj):
         """Check that every action has a unique __name__."""
         names = [name for _, name, _ in obj._get_base_actions()]
         if len(names) != len(set(names)):
+            for item, count in collections.Counter(names).items():
+                if count > 1:
+                    duplicate = str(item)
+                    break
```

Comment:
What if there are multiple objects with repeated  occurrences. Shouldn't you raise error for each repeated occurrence?

Reply:
We're addressing this issue on this thread https://github.com/django/django/pull/12281#discussion_r363682762